OBOFoundry/OBOFoundry.github.io
# Upgrade site to Bootstrap 5

This PR does the following:

1. Massive deletion of unused, vendored Bootstrap 3 and jQuery assets from 2015/2016
2. Massive deletion of unused custom CSS
3. Reorganization of Javascript assets (there was no need to use a maze of includes/imports in Liquid)
4. Upgrade template to use Bootstrap 5

Before merge, I think this warrants some external testing.

## HOW TO TEST

After getting docker running, do the following in your shell

```shell
git clone https://github.com/cthoyt/OBOFoundry.github.io.git@upgrade-bootstrap obofoundry
cd obofoundry
docker run --rm --volume="$PWD:/srv/jekyll" -p 4000:4000 -it jekyll/jekyll:3.5 jekyll serve
```

Finally, navigate to http://localhost:4000 and see how everything works.

## Screenshots

<details>
<summary>Click to see screenshots</summary>

<img width="1597" alt="Screen Shot 2022-10-10 at 01 58 09" src="https://user-images.githubusercontent.com/5069736/194785380-4ac298f1-2671-40db-aaca-7383af6c211a.png">
<img width="1597" alt="Screen Shot 2022-10-10 at 01 58 13" src="https://user-images.githubusercontent.com/5069736/194785382-af14105e-4427-4ae7-b273-fd2c395bbe9b.png">
<img width="1597" alt="Screen Shot 2022-10-10 at 01 58 23" src="https://user-images.githubusercontent.com/5069736/194785385-5ec44d9e-ed25-4138-936b-24f0e581c8ea.png">
<img width="1597" alt="Screen Shot 2022-10-10 at 01 58 55" src="https://user-images.githubusercontent.com/5069736/194785386-8a7aec8a-5043-4a2b-ab17-5c05f0a01248.png">

</details>
